src/react-apps/applications/service-development/src/features/deploy/containers/deployToTestContainer.tsx

```diff
@@ -60,6 +62,7 @@ export interface IDeployToTestContainerProps extends WithStyles<typeof styles> {
   compileStatusUniqueFilenames: [];
   deploymentList: any;
   deployStatus: any;
+  imageVersions: any;
   language: any;
   masterRepoStatus: any;
   repoStatus: any;
```

Comment:
We need to have typings, not suggesting that we set time to do all that now, but maybe create a task for technical debt for typings (if there isn't one already)

Reply:
I've written typings as a refactor task in the epic for "Deployment to several environment".